src/bidiMapper/modules/bluetooth/BluetoothProcessor.ts

```diff
@@ -87,6 +87,8 @@ export class BluetoothProcessor {
   #bluetoothDevices: Map<string, BluetoothDevice>;
   // A map from a characteristic id from CDP to its BluetoothCharacteristic object.
   #bluetoothCharacteristics: Map<string, BluetoothCharacteristic>;
+  // A map from a descriptor id from CDP to its BluetoothDescriptor object.
+  #bluetoothDescriptors: Map<string, BluetoothDescriptor>;
```

Comment:
nit: you can initialize them in-place, and remove initialization from the constructor:
```suggestion
  #bluetoothDevices = new Map<string, BluetoothDevice>();
  // A map from a characteristic id from CDP to its BluetoothCharacteristic object.
  #bluetoothCharacteristics = new Map<string, BluetoothCharacteristic>();
  // A map from a descriptor id from CDP to its BluetoothDescriptor object.
  #bluetoothDescriptors  = new Map<string, BluetoothDescriptor>();
```